### PR TITLE
[README] Updated badge and demo path

### PR DESCRIPTION
**Pull request by lchrusciel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 
 [![License](https://img.shields.io/packagist/l/Sylius/Sylius.svg)](https://packagist.org/packages/sylius/sylius)
 [![Version](https://img.shields.io/packagist/vpre/Sylius/Sylius.svg)](https://packagist.org/packages/sylius/sylius)
-[![Build status on Linux](https://img.shields.io/travis/Sylius/Sylius/master.svg)](http://travis-ci.org/Sylius/Sylius)
+[![Build Status on Linux](https://travis-ci.org/Sylius/SyliusDemo.svg?branch=master)](https://travis-ci.org/Sylius/SyliusDemo)
 [![Scrutinizer Quality Score](https://img.shields.io/scrutinizer/g/Sylius/Sylius.svg)](https://scrutinizer-ci.com/g/Sylius/Sylius/)
 [![Total Downloads](https://poser.pugx.org/sylius/sylius/downloads)](https://packagist.org/packages/sylius/sylius)
 
-**This is Sylius Demo Edition repository deployed to demo.sylius.org.**
+**This is Sylius Demo Edition repository deployed to [demo.sylius.org](https://demo.sylius.com/).**
```
Comment (Zales0123) [inline README.md]: One more thing, it could be `demo.sylius.com`, as well as `docs.sylius.com` below :)

File: README.md [hunk 2]
```diff
@@ -17,7 +21,7 @@ We're using full-stack Behavior-Driven-Development, with [phpspec](http://phpspe
 Documentation
 -------------
 
-Documentation is available at [docs.sylius.org](http://docs.sylius.org).
+Documentation is available at [docs.sylius.com](http://docs.sylius.com).
```
Comment (pamil) [inline README.md]: `http` -> `https`

File: README.md [hunk 3]
```diff
@@ -48,7 +52,7 @@ If you are using the supplied Vagrant development environment, please see the re
 Contributing
 ------------
 
-Would like to help us and build the most developer-friendly eCommerce platform? Start from reading our [Contributing Guide](http://docs.sylius.org/en/latest/contributing/index.html)!
+Would like to help us and build the most developer-friendly eCommerce platform? Start from reading our [Contributing Guide](http://docs.sylius.com/en/latest/contributing/index.html)!
```
Comment (pamil) [inline README.md]: `http` -> `https`

File: README.md [hunk 4]
```diff
@@ -1,12 +1,16 @@
-![Sylius](http://demo.sylius.org/assets/shop/img/logo.png)
+<h1 align="center">
+    <a href="http://sylius.com" target="_blank">
```
Comment (pamil) [inline README.md]: `http` -> `https`

**Comment by pamil** [comment]
Thank you!

**Pull request merged**